sormas-cargoserver/build_deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```

Comment:
I would maybe add an example usage comment at the top like this:


_Execute with `./build_deploy.sh <arg>` , e.g. `./build_deploy.sh fast`_

Reply:
done